packages/common/src/signHelpers.ts

```diff
@@ -0,0 +1,82 @@
+import { getAccountTransactionSignDigest } from './serialization';
+import {
+    AccountTransaction,
+    CredentialSignature,
+    AccountTransactionSignature,
+} from './types';
+import { countSignatures } from './util';
+import * as ed from 'noble-ed25519';
+import { Buffer } from 'buffer/';
+
+export interface AccountSigner {
+    sign(digest: Buffer): Promise<AccountTransactionSignature>;
+    getSignatureCount(): bigint;
+}
+
+export function buildBasicAccountSigner(privateKey: string): AccountSigner {
```

Comment:
Document exported functions, e.g. it is important to state the format of `privateKey`.